### PR TITLE
Unary ops norm_logcdf, norm_derivlogcdf for log CDF of standard normal

### PR DESCRIPTION
**Pull request by mseeger**
* New unary ops

* Improved unit tests for basic unary, binary ops

## Description ##
Adds unary ops norm_logcdf, norm_derivlogcdf for log CDF of standard normal. Basic functionality (see scipy.stats.norm), required to implement Bayesian optimization or expectation propagation.
Also improved unit tests for basic unary, binary ops. Now, comparison to NumPy done for float64 only, then float64 code is compared to float32, float16.

### Essentials ###
- [ ] Passed code style checking (`make lint`)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: docs/api/python/ndarray/ndarray.md [hunk 1]
```diff
@@ -606,6 +606,8 @@ The `ndarray` package provides several classes:
     sign
     gamma
     gammaln
+	norm_logcdf
+	norm_derivlogcdf
```
Comment (piiswrong) [inline docs/api/python/ndarray/ndarray.md]: indentation

File: docs/api/python/symbol/symbol.md [hunk 2]
```diff
@@ -607,6 +607,8 @@ Composite multiple symbols into a new one by an operator.
     sign
     gamma
     gammaln
+	norm_logcdf
```
Comment (piiswrong) [inline docs/api/python/symbol/symbol.md]: same

File: docs/api/python/ndarray/ndarray.md (hunk 1 above)
Comment (mseeger) [reply docs/api/python/ndarray/ndarray.md]: About the name: logcdf is too unspecific.
logcdf_norm, derivlogcdf_norm would be OK.

Was "inspired" by scipy.stats.norm.logcdf.

**Comment by mseeger** [comment]
I can change names to logcdf_normal, derivlogcdf_normal.

**Pull request closed without merging**